app/(onboard)/_components/OnboardWizard.tsx

```diff
@@ -0,0 +1,66 @@
+'use client';
+
+import { useState, useEffect } from 'react';
+import OnboardSteps from '~/app/(onboard)/_components/OnboardSteps/OnboardSteps';
+import CreateAccount from '~/app/(onboard)/_components/OnboardSteps/CreateAccount';
+import ConfigureStudy from '~/app/(onboard)/_components/OnboardSteps/ConfigureStudy';
+import Documentation from '~/app/(onboard)/_components/OnboardSteps/Documentation';
+import { cn } from '~/utils/shadcn';
+import { useRouter, useSearchParams } from 'next/navigation';
+import { checkConfigExpired } from '~/app/actions';
+
+function OnboardWizard() {
+  const searchParams = useSearchParams();
+  const [step, setStep] = useState(searchParams.get('step') ?? '1');
+  const router = useRouter();
+
+  useEffect(() => {
+    checkConfigExpired()
```

Comment:
Maybe we could just leave checking for expired config in the page for this route? Or even the root layout?

Side note about that: given that we want to check that the app is configured, and that the config window isn't expired for basically _every_ request, we should probably think about storing the result of the checkConfigExpired test inside of `cache()`: https://react.dev/reference/react/cache

I'll add this comment there, too.

Reply:
removed check for config expired from this file here- [608d192](https://github.com/complexdatacollective/Fresco/pull/10/commits/608d192cb35a5a253fe09ec999a54fb6765095dd)

other feedback here about caching, moving check to root layout resolved/addressed in other conversation